axelor-human-resource/src/main/java/com/axelor/apps/hr/web/expense/ExpenseController.java

```diff
@@ -274,6 +274,12 @@ public void compute(ActionRequest request, ActionResponse response) {
     response.setValues(expense);
   }
 
+  public void recentExpenseDate(ActionRequest request, ActionResponse response) {
+    Expense expense = request.getContext().asType(Expense.class);
+    expense = expenseServiceProvider.get().updateMoveDateAndPeriod(expense);
```

Comment:
call updateMoveDateAndPeriod() service in existing action-method of compute, no need of new action-method